Formula/twilio.rb

```diff
@@ -3,8 +3,8 @@
 class Twilio < Formula
   desc "unleash the power of Twilio from your command prompt"
   homepage "https://github.com/twilio/twilio-cli"
-  url "https://twilio-cli-prod.s3.amazonaws.com/twilio-v3.0.0/twilio-v3.0.0.tar.gz"
   version "3.0.0"
+  url "https://twilio-cli-prod.s3.amazonaws.com/twilio-v#{version}/twilio-v#{version}.tar.gz"
```

Comment:
How is hardcoding the formula preventing to downgrade cli?

Reply:
+1
We shouldn't be needing these changes, and another Formula should ideally be released `Formula@2.x.x`